charts/catalog/templates/controller-manager-deployment.yaml

```diff
@@ -55,7 +55,7 @@ spec:
         - --service-catalog-api-server-url
         - https://{{ template "fullname" . }}-apiserver
         {{- end }}
-        {{ if .Values.controllerManager.apiserverSkipVerify -}}
+        {{ if and (.Values.controllerManager.apiserverSkipVerify) (not .Values.useAggregator) -}}
```

Comment:
1. aggregation won't work without tls setup, so this doesn't make sense.
2. it's extra steps to generate certs that I'm not sure how to do with helm or can be sure that will be able to be run before helm runs for sure. I'd like to have it, I'm just not sure how.
3. I think this is how you would solve 2, but I don't know how to do this with helm.